EIPS/eip-5773.md

```diff
@@ -0,0 +1,362 @@
+---
+eip: 5773
+title: Multi-Resource Token
+description: An interface for Multi-Resource tokens.
+author: Bruno Škvorc (@Swader), Cicada (@CicadaNCR), Steven Pineda (@steven2308), Stevan Bogosavljevic (@stevyhacker), Jan Turk (@ThunderDeliverer)
+discussions-to: https://ethereum-magicians.org/t/multiresource-tokens/11326
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-10
+requires: 165, 721
+---
+
+## Abstract
+
+The Multi-Resource NFT standard allows for the construction of a new primitive: context-dependent output of multimedia information per single NFT.
```

Comment:
WDYM by primitive?

Reply:
We were referring to a simple, clearly defined, structure

Reply:
```suggestion
The Multi-Resource NFT standard allows for the construction of a new simple, clearly defined structure for context-dependent output of multimedia information per single NFT.
```
"Primitive" is very often associated with language features. I suggest this wording.

Reply:
This makes sense. Thank you 😄